lib/ansible/modules/files/lineinfile.py

```diff
@@ -53,10 +53,10 @@
     version_added: 1.7
     description:
       - The regular expression to look for in every line of the file. For
-        C(state=present), the pattern to replace if found; only the last line
-        found will be replaced. For C(state=absent), the pattern of the line
-        to remove.  Uses Python regular expressions; see
-        U(http://docs.python.org/2/library/re.html).
+        C(state=present), the pattern to replace if found (even if you use insertafter
+        or insertbefore); only the last line found will be replaced.
```

Comment:
This wording is confusing. Did you mean to say "even if you use insertbefore or insertafter, only the last line found will be replaced"? If so, please remove the parenthesis. Thanks.